xrviz/fields.py

```diff
@@ -1,4 +1,5 @@
 import panel as pn
+from packaging.version import Version
```

Comment:
You might want `LooseVersion`, because the version string could contain non-standard parts such as "rc", "beta", or git information.